source/blog/2016-11-28-ember-2-10-released.md

```diff
@@ -1,26 +1,58 @@
 ---
-title: Ember 2.10.0 and 2.11.0-beta.1 Release
+title: Ember 2.10 and 2.11 Beta Released
 author: Godfrey Chan, Brendan McLoughlin, Nathan Hammond
 tags: Releases
 ---
 
+Today, the Ember project is releasing Ember.js, Ember Data and Ember CLI
+version 2.10.0.
+
+This also kicks off the 2.11 beta cycle for all sub-projects. We enourage our
+community (especially addon authors) to help test these beta builds and report
+any bugs before they are published as a final release in six weeks' time. (The
+[ember-try](https://github.com/ember-cli/ember-try) addon is a great way to do
+that.)
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
 ## Ember.js
 
+Ember.js is the core framework for building ambitious web applications.
 
+### Changes in Ember.js 2.10
 
+...
 
-## Ember Data
+For more details on the changes in Ember.js 2.10, please review the
+[Ember.js 2.10.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.10.0).
+
+### Upcoming changes in Ember.js 2.11
```

Comment:
I'm a bit worried this might frame as "they're definitely coming", but in the Data section there's already an exception to the rule. am I reading too much into it?